src/challenges.js

```diff
@@ -1,47 +1,111 @@
 // Desafio 1
-function compareTrue() {
-  // seu código aqui
+function compareTrue(param1, param2) {
+  return param1 && param2;
 }
 
 // Desafio 2
-function calcArea() {
-  // seu código aqui
+function calcArea(base, height) {
+  let totalArea = ((base * height) / 2);
```

Comment:
@vitorguima o que você acha de remover os parenteses que circundam a operação e não fazem diferença?

```suggestion
  let totalArea = (base * height) / 2;
```

Reply:
show, obrigado pela sugestão, ajuste realizado! :-)